### PR TITLE
Add base configuration to make the chat work

### PR DESCRIPTION
**Pull request by Klynger**
#### What is the purpose of this pull request?

As the title says.

#### How should this be manually tested?

[Workspace](https://klynger--gympassus.myvtex.com/)

#### Screenshots or example usage

image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [X] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: pixel/head.html [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+    zE(function () {
+      $zopim(function() {
+        zE('webWidget', 'setLocale', curLocale)
+        $zopim.livechat.sendVisitorPath({
+          url: urlPath,
+          title: titlePath
+        })
+      })
+
+      zE.identify({
```
Comment (klzns) [inline pixel/head.html]: Use session promise to fill in those fields async
```js
__RENDER_8_SESSION__.sessionPromise.then((session) => console.log(session))
```

File: pixel/head.html [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+        name: '',
+        email: ''
+      })
+    })
+
+    var analytics = '{{ settings.analytics }}' === 'true'
+    var webWidget = JSON.parse(window.decodeURIComponent('{{ settings.webWidget }}'))
+    var helpCenterSuppress =  webWidget.helpCenter.suppress
+    var departments = webWidget.chat.departments
```
Comment (klzns) [inline pixel/head.html:48]: Check if the department is a number or string and change its type accordingly

**Review by klzns — CHANGES_REQUESTED** [review]

File: pixel/head.html [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+        })
+      })
+
+      zE.identify({
+        name: '',
+        email: ''
+      })
+    })
+
+    var analytics = '{{ settings.analytics }}' === 'true'
```
Comment (iaronaraujo) [inline pixel/head.html]: can't this be only `{{ settings.analytics }}` ?

**Review by iaronaraujo — CHANGES_REQUESTED** [review]

File: pixel/head.html (hunk 3 above)
Comment (Klynger) [reply pixel/head.html]: No, this is always a string (sometimes empty string sometimes the value as a string `"true"` or `"false"`)

File: pixel/head.html [hunk 4]
```diff
@@ -15,13 +25,21 @@
         })
       })
 
-      zE.identify({
-        name: '',
-        email: ''
-      })
+      if (window.__RENDER_8_SESSION__ && window.__RENDER_8_SESSION__.sessionPromise) {
+        window.__RENDER_8_SESSION__.sessionPromise.then(function (session) {
+          var userEmail = path(['response', 'namespaces', 'authentication', 'storeUserEmail', 'value'], session)
```
Comment (klzns) [inline pixel/head.html]: Use profile instead
image

File: pixel/head.html [hunk 5]
```diff
@@ -2,9 +2,19 @@
 <script>
   window.addEventListener('load', function() {
     var urlPath = document.URL
-    var titlePathPreffix = '{{ settings.titlePreffix }}'
+    var titlePathPreffix = window.decodeURIComponent('{{ settings.titlePreffix }}')
     var titlePath = titlePathPreffix + document.querySelector('head > title').innerText
     var curLocale = document.querySelector('html').lang
+
+    function path(pathArray = [], obj) {
```
Comment (klzns) [inline pixel/head.html]: Use `window.R.path` instead

**Review by klzns — APPROVED** [review]

**Review by iaronaraujo — APPROVED** [review]

**Pull request merged**